tests/test_detect_target.py

```diff
@@ -58,85 +55,83 @@ def image_zidane():
     assert zidane_image is not None
     yield zidane_image
 
+@pytest.fixture()
+def expected_bus():
+    """
+    Load expected bus image.
+    """
+    expected_bus = np.loadtxt(BOUNDING_BOX_BUS_PATH)
+    assert expected_bus is not None
+    yield expected_bus
```

Comment:
Return a type of `detections_and_time.DetectionsAndTime` , not `np.ndarray` . The fixture should handle all the nasty setup part so that the test doesn't need to do data manipulation. This will mean modifying all of the tests (again).

You can also modify the image timestamp directly in the image loading fixtures and these fixtures (e.g. setting it to 0 or something), since the timestamp is not relevant to this test.